### PR TITLE
Update FR in translation.json

### PR DESCRIPTION
**Pull request by mooondark**

### CODE REVIEW

File: translations.json [hunk 1]
```diff
@@ -165,22 +167,25 @@
...
             "en": "Status:",
-            "fr-FR": "Etat",
+            "fr-FR": "Etat :",
             "ja-JP": "ステータス :",
             "ko-KR": "상태:",
             "zh-CN": "状态："
         },
         "label4.Text": {
             "en": "Overlay:",
+            "fr-FR": "Overlay :',
```
Comment (ngld) [inline translations.json]: The quote at the end should be `"`, not `'`.

File: translations.json [hunk 2]
```diff
@@ -165,22 +167,25 @@
...
             "zh-CN": "悬浮窗："
         },
         "label5.Text": {
             "en": "URL:",
+            "fr-FR": "URL :',
```
Comment (ngld) [inline translations.json]: Is the space here (and in the previous translation) intentional?

**Review by ngld — CHANGES_REQUESTED** [review]

File: translations.json (hunk 2 above)
Comment (mooondark) [reply translations.json]: yeah in french there's always a space before if the symbol have 2 parts like in : ! ; ?

File: translations.json (hunk 1 above)
Comment (mooondark) [reply translations.json]: Yep !

**Review by mooondark — COMMENTED** [review]
Modified as requested

**Comment by ngld** [comment]
Thanks!

**Pull request merged**